### PR TITLE
Remove CLA

### LINKED ISSUE #591: Add CLA-Bot
Currently our CLA is just a text within the Pull Request. So far I (in me aka @aepfli) can not remember that i really checked when merging if the user agreed upon it.

Therefore i suggest to use https://github.com/apps/cla-bot which can automate this with an additional check. (as suggested by @beatngu13 )

As this is a change/application which needs certain permissions, we should discuss it, and decide upon feasibility - also i am not sure if somebody else besides @nipafx can do it.

### PR DESCRIPTION
**Pull request by beatngu13**
Closes #591.

Proposed commit message:

```
Remove CLA (#591 / #610)

[GitHub's terms of
service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#6-contributions-under-repository-license) already requires users to agree to our license and that they have
the right to license their contribution(s) under those terms. Therefore,
having a dedicated CLA is not necessary.

Closes: #591
PR: #610
```

---
**PR checklist**

The following checklist shall help the PR's author, the reviewers and maintainers to ensure the quality of this project.
It is based on our contributors guidelines, especially the ["writing code" section](https://github.com/junit-pioneer/junit-pioneer/blob/main/CONTRIBUTING.md#writing-code).
It shall help to check for completion of the listed points.
If a point does not apply to the given PR's changes, the corresponding entry can be simply marked as done. 

Documentation (general)
* [ ] There is documentation (Javadoc and site documentation; added or updated)
* [ ] There is implementation information to describe _why_ a non-obvious source code / solution got implemented
* [ ] Site documentation has its own `.adoc` file in the `docs` folder, e.g. `docs/report-entries.adoc`
* [ ] Site documentation in `.adoc` file references demo in `src/demo/java` instead of containing code blocks as text
* [ ] Only one sentence per line (especially in `.adoc` files)
* [ ] Javadoc uses formal style, while sites documentation may use informal style

Documentation (new extension)
* [ ] The `docs/docs-nav.yml` navigation has an entry for the new extension
* [ ] The `package-info.java` contains information about the new extension

Code
* [ ] Code adheres to code style, naming conventions etc.
* [ ] Successful tests cover all changes
* [ ] There are checks which validate correct / false usage / configuration of a functionality and there are tests to verify those checks
* [ ] Tests use [AssertJ](https://assertj.github.io/doc/) or our own [PioneerAssert](https://github.com/junit-pioneer/junit-pioneer/blob/main/CONTRIBUTING.md#assertions) (which are based on AssertJ)

Contributing
* [x] A prepared commit message exists
* [ ] The list of contributions inside `README.md` mentions the new contribution (real name optional) 

---

I hereby agree to the terms of the [JUnit Pioneer Contributor License Agreement](https://github.com/junit-pioneer/junit-pioneer/blob/main/CONTRIBUTING.md#junit-pioneer-contributor-license-agreement).

### CODE REVIEW

**Review by aepfli — APPROVED** [review]

**Comment by beatngu13** [comment]
Sorry, this is a change probably _all_ maintainers should approve.

**Review by Michael1993 — APPROVED** [review]

**Review by Bukama — APPROVED** [review]

**Pull request merged**
